tools/sklearn/feature_selection.xml

```diff
@@ -19,19 +19,29 @@ import sys
 import json
 import pandas
 import pickle
+import ast
```

Comment:
is this still needed?

Reply:
This one is used in the place you commented on blow.